snowshu/configs.py

```diff
@@ -17,6 +17,7 @@
 DOCKER_REMOUNT_DIRECTORY = 'snowshu_replica_data'
 DOCKER_TARGET_PORT = 9999
 DOCKER_WORKING_DIR = Path('/app').as_posix()
+POSTGRES_IMAGE = 'postgres:12'
```

Comment:
Since we don't need this anywhere else, it might be worth leaving the postgres image in that specific adapter

Reply:
That is used in some integration tests, so I think it's better this way than engineering a way to correctly inject target adapter everywhere.